packages/web/app/src/components/ui/empty-list.tsx

```diff
@@ -43,13 +48,61 @@ export const noSchema = (
   />
 );
 
-export const noSchemaVersion = (
-  <EmptyList
-    title="Hive is waiting for your first schema"
-    description="You can publish a schema with Hive CLI and Hive Client"
-    docsUrl="/features/schema-registry#publish-a-schema"
-  />
-);
+export const NoSchemaVersion = ({
+  projectType = null,
+  recommendedAction = 'none',
+}: {
+  projectType: ProjectType | null;
+  recommendedAction: 'publish' | 'check' | 'none';
+}): ReactElement => {
+  let children: ReactElement | null = null;
+  if (recommendedAction !== 'none') {
+    const isDistributed =
+      projectType === ProjectType.Federation || projectType === ProjectType.Stitching;
+
+    if (recommendedAction === 'check') {
+      children = (
+        <>
+          <div className="flex w-full justify-center py-2 text-xs text-gray-500">
+            It's recommended to check that the schema is valid and compatible with the state of the
+            registry before publishing.
+          </div>
+          <div className="flex w-full justify-center">
+            <Code>
+              {`hive schema:check ${isDistributed ? '--service <service-name> --url <url> ' : ''}<path/schema.graphql>`}
```

Comment:
Now that we have the `--target` parameter mandatory for the new access tokens, we should probably also include it here? 🤔 

```
hive schema:check --target "org/project/target ...."
```

https://the-guild.dev/graphql/hive/docs/migration-guides/organization-access-tokens#performing-actions-with-the-cli